conf/pam_conv1/pam_conv_y.y

```diff
@@ -157,11 +159,19 @@ path
 : TOK {
     /* XXX - this could be used to check if file present */
     $$ = strdup(yytext);
+    if ($$ == NULL) {
+	yyerror("failed to duplicate path");
+	exit(1);
+    }
```

Comment:
Likewise.

Reply:
Moved it with the other strdup check into its own commit.